### DISCUSSION TITLE
Vacation Notice: June 23 ~ July 15

### DISCUSSION THREAD

**Discussion by Tao-VanJS** [vanjs-org/van#61] [Announcements]
Hi Fellow **VanJS**ers,

I will be traveling from June 23 to July 15, with no access to my computer. Most likely I can only respond to discussion posts right after I'm back.

Meanwhile, **VanJS** 1.0.0 is planned to be released shortly after my coming back. Stay tuned :-)

Thank you for your understanding!

**Comment by artydev** [comment]
Have a nice rest Tao :-)
